Get rid of allow + -ing (#5045)
DevExpress/devextreme-documentation: concepts/60 Themes and Styles/30 Icons/60 SVG Icons.md

            icon: myIcon
        });

> [important]The SVG format allows to run executable code that might be malicious. We strongly recommend that you use SVG icons only from trusted sources.

Reviewer: It's redundant to say both "run" and "executable". If it's executable, you can run it. If you can run it, it's executable. 

```suggestion
[important]The SVG format can contain executable code that might be malicious. We strongly recommend that you use SVG icons only from trusted sources.
```
